src/usr/local/www/services_dhcp_edit.php

```diff
@@ -385,8 +389,14 @@ function staticmaps_sort($ifgui) {
 
 // Get our MAC address
 $ip = $_SERVER['REMOTE_ADDR'];
-$mymac = `/usr/sbin/arp -an | grep '('{$ip}')' | cut -d" " -f4`;
-$mymac = str_replace("\n", "", $mymac);
+
+if ((ipVersion($ip))=='4') {
```

Comment:
is_ipaddr($ip) will return 4 if it's IPv4, 6 if it's IPv6 or false if it's not a valid IP address. You don't need to add this ipVersion() function here